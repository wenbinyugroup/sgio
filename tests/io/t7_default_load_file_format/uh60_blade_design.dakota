# uh60_blade_design.dakota

environment
  output_file = 'uh60_blade_design.out'
  write_restart = 'uh60_blade_design.rst'
  error_file = 'uh60_blade_design.err'
  tabular_data
    tabular_data_file = 'uh60_blade_design_tabular.dat'
  results_output
    results_output_file = 'uh60_blade_design_results'


method
  soga
    max_iterations =  100
    max_function_evaluations =  20000
    population_size =  200
    seed =  1027
    print_each_pop


model
  single

variables
  active = design

  continuous_design = 11
    descriptors = 'wl_a2_sta1'  'wl_a2_stam'  'wl_a2_sta4'  'wt_a2_sta1'  'wt_a2_stam'  'wt_a2_sta4'  'pnsmc_a2_sta1'  'pnsmc_a2_stam'  'pnsmc_a2_sta4'  'rnsm'  'r_stam'
    upper_bounds = 0.9  0.9  0.9  0.7  0.7  0.7  0.985  0.985  0.985  0.01  0.8
    lower_bounds = 0.8  0.8  0.8  0.5  0.5  0.5  0.95  0.95  0.95  0.001  0.3

  discrete_design_range = 15
    descriptors = 'mi_spar_1'  'mi_le'  'mi_te'  'np_spar_seg1'  'np_spar_seg2'  'np_le_seg1'  'np_le_seg2'  'np_te_seg1'  'np_te_seg2'  'fo_spar_1'  'fo_spar_2'  'fo_spar_3'  'fo_spar_4'  'fo_le'  'fo_te'
    upper_bounds = 4  4  4  20  20  20  20  20  20  90  90  90  90  90  90
    lower_bounds = 1  1  1  1  1  1  1  1  1  -90  -90  -90  -90  -90  -90



interface
  analysis_driver = 'python run.py uh60_blade.yml 1'
    fork
      parameters_file =  'input.in'
      results_file =  'output.out'
      file_save
      work_directory
        directory_tag
        directory_save
        named =  'evals/eval'
        copy_file =  'run.py'  'uh60_blade.yml'  'design/*'  'scripts/*'
  failure_capture
    recover =  1e12  1e12  1e12  1e12


responses
  descriptors =  'diff_gj'  'diff_eiyy'  'diff_eizz'  'sr_min'
  objective_functions = 3
    sense =  'min'  'min'  'min'
    weights =  0.5  0.8  0.8
    nonlinear_inequality_constraints = 1
      upper_bounds =  1e12
      lower_bounds =  1
  no_gradients
  no_hessians


