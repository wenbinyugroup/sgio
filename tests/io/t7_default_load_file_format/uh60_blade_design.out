Writing new restart file 'uh60_blade_design.rst'.

>>>>> Executing environment.

>>>>> Running soga iterator.

---------------------
Begin Evaluation    1
---------------------
Parameters for evaluation 1:
                      8.0014648885e-01 wl_a2_sta1
                      8.6678670614e-01 wl_a2_stam
                      8.0024414808e-01 wl_a2_sta4
                      6.4120303964e-01 wt_a2_sta1
                      6.6954863124e-01 wt_a2_stam
                      5.0837427900e-01 wt_a2_sta4
                      9.8402371288e-01 pnsmc_a2_sta1
                      9.5110339671e-01 pnsmc_a2_stam
                      9.7234778893e-01 pnsmc_a2_sta4
                      9.5976134526e-03 rnsm
                      6.0556657613e-01 r_stam
                                     2 mi_spar_1
                                     3 mi_le
                                     3 mi_te
                                    13 np_spar_seg1
                                    18 np_spar_seg2
                                    17 np_le_seg1
                                    16 np_le_seg2
                                    11 np_te_seg1
                                    10 np_te_seg2
                                   -10 fo_spar_1
                                    82 fo_spar_2
                                     8 fo_spar_3
                                   -48 fo_spar_4
                                    44 fo_le
                                   -53 fo_te

blocking fork: python run.py uh60_blade.yml 1 input.in output.out
Failure captured: recovering with specified function values for evaluation 1.

Active response data for evaluation 1:
Active set vector = { 1 1 1 1 }
                      1.0000000000e+12 diff_gj
                      1.0000000000e+12 diff_eiyy
                      1.0000000000e+12 diff_eizz
                      1.0000000000e+12 sr_min



---------------------
Begin Evaluation    2
---------------------
Parameters for evaluation 2:
                      8.0101321451e-01 wl_a2_sta1
                      8.6067690054e-01 wl_a2_stam
                      8.3944517350e-01 wl_a2_sta4
                      6.1863155004e-01 wt_a2_sta1
                      6.8570513016e-01 wt_a2_stam
                      6.5558946501e-01 wt_a2_sta4
                      9.5747703482e-01 pnsmc_a2_sta1
                      9.7812967315e-01 pnsmc_a2_stam
                      9.6031403546e-01 pnsmc_a2_sta4
                      7.1214941862e-03 rnsm
                      5.6099429304e-01 r_stam
                                     4 mi_spar_1
                                     4 mi_le
                                     2 mi_te
                                    13 np_spar_seg1
                                    10 np_spar_seg2
                                    12 np_le_seg1
                                     1 np_le_seg2
                                    15 np_te_seg1
                                    18 np_te_seg2
                                   -72 fo_spar_1
                                     0 fo_spar_2
                                    78 fo_spar_3
                                     5 fo_spar_4
                                   -77 fo_le
                                    59 fo_te

blocking fork: python run.py uh60_blade.yml 1 input.in output.out

Active response data for evaluation 2:
Active set vector = { 1 1 1 1 }
                      2.2550956751e+01 diff_gj
                      1.5755609554e+01 diff_eiyy
                      4.7906753293e+00 diff_eizz
                      1.0456827057e-01 sr_min



---------------------
Begin Evaluation    3
---------------------
Parameters for evaluation 3:
                      8.0119937742e-01 wl_a2_sta1
                      8.3542588580e-01 wl_a2_stam
                      8.4670552690e-01 wl_a2_sta4
                      6.4152043214e-01 wt_a2_sta1
                      6.5128025147e-01 wt_a2_stam
                      5.5534226508e-01 wt_a2_sta4
                      9.5985366375e-01 pnsmc_a2_sta1
                      9.6302392651e-01 pnsmc_a2_stam
                      9.8074663533e-01 pnsmc_a2_sta4
                      3.2080446791e-03 rnsm
                      4.6322824793e-01 r_stam
                                     3 mi_spar_1
                                     1 mi_le
                                     2 mi_te
                                    13 np_spar_seg1
                                     3 np_spar_seg2
                                    20 np_le_seg1
                                     7 np_le_seg2
                                    17 np_te_seg1
                                     1 np_te_seg2
                                   -53 fo_spar_1
                                    -7 fo_spar_2
                                    -9 fo_spar_3
                                   -88 fo_spar_4
                                    70 fo_le
                                     2 fo_te

blocking fork: python run.py uh60_blade.yml 1 input.in output.out

Active response data for evaluation 3:
Active set vector = { 1 1 1 1 }
                      2.2376478359e+01 diff_gj
                      1.3065574617e+01 diff_eiyy
                      1.7946219997e+00 diff_eizz
                      1.1240173549e-01 sr_min



---------------------
Begin Evaluation    4
---------------------
Parameters for evaluation 4:
                      8.0140079958e-01 wl_a2_sta1
                      8.6338694418e-01 wl_a2_stam
                      8.2909939879e-01 wl_a2_sta4
                      6.3781548509e-01 wt_a2_sta1
                      5.2562334056e-01 wt_a2_stam
                      5.3720206305e-01 wt_a2_sta4
                      9.6968169195e-01 pnsmc_a2_sta1
                      9.5901196326e-01 pnsmc_a2_stam
                      9.7721427046e-01 pnsmc_a2_sta4
                      7.0099795526e-03 rnsm
                      4.7227698599e-01 r_stam
                                     3 mi_spar_1
                                     4 mi_le
                                     2 mi_te
                                    13 np_spar_seg1
                                    14 np_spar_seg2
                                    13 np_le_seg1
                                    19 np_le_seg2
                                    13 np_te_seg1
                                     6 np_te_seg2
                                    29 fo_spar_1
                                   -82 fo_spar_2
                                   -61 fo_spar_3
                                   -11 fo_spar_4
                                    74 fo_le
                                    13 fo_te

blocking fork: python run.py uh60_blade.yml 1 input.in output.out

Active response data for evaluation 4:
Active set vector = { 1 1 1 1 }
                      9.3637703374e+00 diff_gj
                      4.1844551988e+00 diff_eiyy
                      1.0259783405e+00 diff_eizz
                      3.6230169309e-02 sr_min



---------------------
Begin Evaluation    5
---------------------
Parameters for evaluation 5:
                      8.0148930326e-01 wl_a2_sta1
                      8.3254799036e-01 wl_a2_stam
                      8.5199133274e-01 wl_a2_sta4
                      6.8610797449e-01 wt_a2_sta1
                      5.1987975707e-01 wt_a2_stam
                      6.4874111148e-01 wt_a2_sta4
                      9.7795983764e-01 pnsmc_a2_sta1
                      9.7963469344e-01 pnsmc_a2_stam
                      9.5110126041e-01 pnsmc_a2_sta4
                      1.7869197668e-03 rnsm
                      7.0142521439e-01 r_stam
                                     2 mi_spar_1
                                     4 mi_le
                                     2 mi_te
                                     4 np_spar_seg1
                                    16 np_spar_seg2
                                    19 np_le_seg1
                                    16 np_le_seg2
                                    12 np_te_seg1
                                    18 np_te_seg2
                                    43 fo_spar_1
                                    10 fo_spar_2
                                   -10 fo_spar_3
                                    73 fo_spar_4
                                   -46 fo_le
                                   -24 fo_te

blocking fork: python run.py uh60_blade.yml 1 input.in output.out

Dakota caught signal 2
